modules/auth/proc-adding-azure-as-an-authentication-provider.adoc

```diff
@@ -0,0 +1,83 @@
+
+*Redirect URI*:: Enter the backend authentication URI set in {product-short}: `pass:c,a,q[http://_<APP_FQDN>_/api/auth/microsoft/handler/frame]`
+*Front-channel logout URL*:: Leave blank.
+*Implicit grant and hybrid flows*:: All unchecked.
+
+.. On the *API permissions* tab, click *Add Permission*, then add the following *Delegated permission* for the
+*Microsoft Graph API*:
+
+`email`:::
+`offline_access`:::
+`openid`:::
+`profile`:::
+`User.Read`:::
+(Optionally)::: Optional custom scopes of the Microsoft Graph API that you define also in the {product-short} configuration (`app-config-rhdh.yaml`).
+
+.. (Optionally) Your company might require you to grant admin consent for these permissions.
+Even if your company does not require link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/user-admin-consent-overview[admin consent], you might do so as it means users do not need to individually consent the first time they access backstage.
+To grant admin consent, a directory admin must go to this page and click the *Grant admin consent for COMPANY NAME* button.
+
+.. Go to the *Certificates & Secrets* page, then the *Client secrets* tab, and create a new client secret.
+Save the *Client secret* for the next step.
+
+. Add your Gitlab credentials in your {product-short} secrets.
++
+Edit your {product-short} secrets, such as `secrets-rhdh`.
++
+Add the following key/value pairs:
++
+`AUTH_AZURE_CLIENT_ID`:: Enter the *Application ID* that you generated on Microsoft Azure.
+`AUTH_AZURE_CLIENT_SECRET`:: Enter the *Client secret* that you generated on Microsoft Azure.
+`AUTH_AZURE_TENANT_ID`:: Enter your *Tenant ID* on Microsoft Azure.
+
+. Set up the Gitlab authentication provider in your {product-short} custom configuration.
++
+Edit your custom {product-short} config map, such as `app-config-rhdh`.
++
+In the `app-config-rhdh.yaml` content, add the `microsoft` provider configuration under the root `auth` configuration, and enable the `microsoft` provider for sign-in:
++
+.`app-config-rhdh.yaml` fragment
+[source,yaml,subs="+quotes,+attributes"]
+----
+auth:
+  environment: development
+  providers:
+    microsoft:
+      development:
+        clientId: ${AUTH_AZURE_CLIENT_ID}
+        clientSecret: ${AUTH_AZURE_CLIENT_SECRET}
+        tenantId: ${AUTH_AZURE_TENANT_ID}
+        domainHint: ${AUTH_AZURE_TENANT_ID} # <1>
+        # additionalScopes: # <2>
+          # - Mail.Send
+signInPage: microsoft # <3>
+----
++
+<1> Optionally, when your application is not multi tenant,
+to reduce login friction for users with accounts in multiple tenants
+by automatically filtering away accounts from other tenants:
+uncomment, and enter the tenant id.
+Typically, the same as `tenantId`.
+Leave blank if your app registration is multi tenant.
+See link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/home-realm-discovery-policy[Home Realm Discovery].
```

Comment:
```suggestion
<1>  Optional for single-tenant applications. You can reduce login friction for users with accounts in multiple tenants by automatically filtering out accounts from other tenants. If you want to use this parameter for a single-tenant application, uncomment and enter the tenant ID. If your application registration is multi-tenant, leave this parameter blank. For more information, see link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/home-realm-discovery-policy[Home Realm Discovery].
```

Reply:
This new wording might need to be double-checked by a SME, and I'm not exactly sure what _uncomment_ means, but I think this revision more clearly communicates our intended message.